src/onegov/agency/templates/macros.pt

```diff
@@ -263,9 +263,11 @@
                 <a href="${request.link(agency)}" i18n:translate="">Show agency</a>
             </div>
             <ul class="memberships fa-ul" tal:condition="memberships">
-               <li tal:repeat="membership memberships" class="person"><span class="fa-li"><i class="fa fa-user-circle"></i></span>
-                   <b><a href="${request.link(membership.person)}">${membership.person.title}</a></b><tal:block tal:condition="membership.title">, <a href="${request.link(membership.person)}">${membership.title}</a></tal:block>
-               </li>
+                <tal:block tal:repeat="membership memberships" >
+                    <li tal:condition="(membership.access == 'public' and membership.published and membership.person.access == 'public' and membership.person.published) or request.is_logged_in" class="person"><span class="fa-li"><i class="fa fa-user-circle"></i></span>
```

Comment:
`membership.person` might be `None` in some edge case... It's probably better to check:
logged in OR (membership access AND membership public AND (not person OR (person access AND person public)))